### PR TITLE
two test cases using WVFPDP

### PR DESCRIPTION
**Pull request by tskille**
both test cases include one producer with pressure support from an analytical aquifer. 

```
WVFPDP-01 : item2 equal to -20.0, item 3 defaulted 
WVFPDP-01 : item2 equal to -20.0 and item 3 equal to 0.75
```

Keyword WVFPDP is currently not supported in OPM FLow.

### CODE REVIEW

File: vfp/WVFPDP-02.DATA [hunk 1]
```diff
@@ -0,0 +1,333 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2023 Equinor
+
+-- This test case is testing keyword WVFPDP (adjusting bottom hole pressure from vfp). This case includes 
+-- one producer with aquifer support. The producer is using artificial lift last 60 days of the simulation.  
+\-- Item 2 in keyword WVFPDP is set to -20.0 bars and item 3 (loss scaling factor) is set to 0.75
```
Comment (bska) [inline vfp/WVFPDP-02.DATA:10]: This backslash character at the beginning of the line looks a little strange to me.  Is that really supposed to be a valid comment character?

File: vfp/WVFPDP-01.DATA [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+/
+
+WLPR
+ /
+
+WGIR
+/
+ 
+FPR
+
```
Comment (bska) [inline vfp/WVFPDP-01.DATA:270]: Would it make sense to add a few aquifer-related vectors too, e.g., `AAQP` and/or `AAQR`?

**Review by bska — COMMENTED** [review]
I did not review the VFP table, so I'll just assume that it covers a reasonable range of inputs and has a reasonable set of outputs.  Apart from a possible misprint in the header of `-02`, this all looks good to me.  We might want to add a few more summary vectors though.

File: vfp/WVFPDP-02.DATA (hunk 1 above)
Comment (tskille) [reply vfp/WVFPDP-02.DATA]: tanks, my bad. I keystroke that ended up in a wrong window. 

This is not a valid line and Flow will give an error message 
```
String \ not formatted as valid keyword
```

File: vfp/WVFPDP-01.DATA (hunk 2 above)
Comment (tskille) [reply vfp/WVFPDP-01.DATA:283]: Yes, good point. I've also added gas lift injection rate since this is also used for the producer.

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  I'll merge this into the master branch now.

**Pull request merged**